docs/install/manual-install.md

```diff
@@ -128,11 +128,45 @@ As well as installing dependencies, this script compiles our CSS (using
 `bin/make_css`), installs the database schema (using `bin/update-schema`),
 and compiles any translation `.mo` files (using `commonlib/bin/gettext-makemo`).
 
+#### macOS troubleshooting
+
+Some Perl modules may fail to install without certain packages and paths in place.
+
+**Image::PNG::QRCode** requires the `libpng` package.
+
+For any **SSL-related modules**, make sure `openssl` is installed. If using homebrew:
+
+    $ brew install --cask openssl
+
+Then you may need to create symlinks to folders Perl modules will recognise:
+
+    $ sudo mkdir /usr/local/lib
+
+    $ sudo ln -s /opt/homebrew/opt/openssl@1.1/lib/libssl.1.1.dylib /usr/local/lib/libssl.dylib
+
+    $ sudo ln -s /opt/homebrew/opt/openssl@1.1/lib/libcrypto.1.1.dylib /usr/local/lib/libcrypto.dylib
```

Comment:
Looks like the convention in this file is to use the `{% highlight bash %}` tag for syntax highlighting and have no spaces between lines.

```suggestion
{% highlight bash %}
$ sudo mkdir /usr/local/lib
$ sudo ln -s /opt/homebrew/opt/openssl@1.1/lib/libssl.1.1.dylib /usr/local/lib/libssl.dylib
$ sudo ln -s /opt/homebrew/opt/openssl@1.1/lib/libcrypto.1.1.dylib /usr/local/lib/libcrypto.dylib
{% endhighlight %}
```

Reply:
I'll change this to be consistent with the rest of the doc, but I am not sure why this pattern is used, as it does not work in my VSCode preview, nor in GitHub (which I thought supported extended MD syntax) - is it even part of extended MD syntax?

Reply:
It's Jekyll highlighting, see e.g. https://gist.github.com/roachhd/e434c1481e74841749d7 point 1. It works on https://fixmystreet.org/install/manual-install/ I think? :)